setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='simplify-debts',
+    version='1.0',
+    description='Tool to perform mutual debt simplification',
+    url='https://github.com/ric2b/Mutual_Debt_Simplification',
+    license='MIT',
+    author='',
+    author_email='',
+
+    packages=find_packages(exclude=['*.tests']),
+
+    install_requires=['graphviz==0.8'],
+
+    extras_require={
+        'test': ['unittest'],
+    },
+
+    entry_points={
```

Comment:
really cool :)

Reply:
I had some trouble with getting it to work because of Path issues on Ubuntu, it's fixed by adding `export PATH=~/.local/bin:$PATH` to ~/.bashrc and running `source ~/.bashrc` to activate it in the current session

Reply:
I had the same issue, but I was using z-shell. I though that was the reason for '~/.local/bin' not be included in PATH. I don't understand why is this not included by default, since all python console scripts are stored there, when installed in the user directory.